tests/functional/test_fast_deps.py

```diff
@@ -136,3 +164,188 @@ def test_hash_mismatch_existing_download_for_metadata_only_wheel(
         hash_file(str(idna_wheel))[0].hexdigest()
         == "b97d804b1e9b523befed77c48dacec60e6dcb0b5391d57af6a65a312a90648c0"
     )
+
+
+@mark.parametrize("range_handler", list(RangeHandler))
```

Comment:
Do we plan to add more handlers here in the future?

Reply:
This actually converts the `RangeHandler` enum into a list of all its values!

```bash
# ...
tests/functional/test_fast_deps.py::test_download_range[RangeHandler.Always200OK]
# ...
```